src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderStatic.cs

```diff
@@ -256,5 +256,12 @@ public Column Create()
             }
         }
     }
-}
 
+    public static class LocalPathReader
```

Comment:
>LocalPathReader [](start = 24, length = 15)

I don't think it belongs to `TextLoaderStatic`.

Reply:
so... where does it belong :)

---
In reply to: [226074992](https://github.com/dotnet/machinelearning/pull/1281#discussion_r226074992) [](ancestors = 226074992)

Reply:
Actually, I just added an instance method to `TextLoader` :)

---
In reply to: [226082986](https://github.com/dotnet/machinelearning/pull/1281#discussion_r226082986) [](ancestors = 226082986,226074992)

Reply:
See my MLContext PR

---
In reply to: [226084273](https://github.com/dotnet/machinelearning/pull/1281#discussion_r226084273) [](ancestors = 226084273,226082986,226074992)